wemake_python_styleguide/violations/complexity.py

```diff
@@ -518,8 +518,8 @@ class TooManyBaseClassesViolation(ASTViolation):
     Reasoning:
         It is almost never possible to navigate
         to the desired method of a parent class
-        when you need it with multiple mixin.
-        It is hard to understand mro and super calls.
+        when you need it with multiple mixins.
+        It is hard to understand 'mro' and 'super' calls.
```

Comment:
Please, use double `` to wrap `mro` and `super`. Like here: https://github.com/wemake-services/wemake-python-styleguide/blob/ffc72dc3fe0a04e36527f9e06ab9b1eccfff4d92/wemake_python_styleguide/violations/complexity.py#L169